src/internal/components/Tabs/Tab.tsx

```diff
@@ -0,0 +1,39 @@
+import { background, cn, color, text } from '@/styles/theme';
+import { useCallback } from 'react';
+import { useTabsContext } from './Tabs';
+
+type TabReact = {
+  value: string;
+  children: React.ReactNode;
+  className?: string;
+  ariaLabel?: string;
```

Comment:
Can we follow standard html attributes? That's how Paul's done them in the internal primitives / is standard in React (e.g. `HTMLAttributes<HTMLButtonElement>>`)

```suggestion
  aria-label?: string;
```